src/AppKit/NSBezierPath.cs

```diff
@@ -110,5 +110,15 @@ public unsafe void AppendPathWithGlyphs (uint[] glyphs, NSFont font)
 				_AppendPathWithGlyphs ((IntPtr)ptr, glyphs.Length, font);
 		}
 
+		public unsafe void AppendBezierPathWithCGGlyphs (uint[] glyphs, NSFont font)
```

Comment:
There are "new" glyph APIs and Apple deprecated a whole mess of "old" ones. The diff was really ugly when I was parsing through it, and it appears I missed some.